### ISSUE TITLE
JRE 1.8 support broken in 0.3.2

### ISSUE THREAD

**Issue by guersam** [lomigmegard/akka-http-cors#44]
0.3.2 causes an `UnsupportedClassVersionError` in JRE 1.8:

```
java.lang.UnsupportedClassVersionError: ch/megard/akka/http/cors/javadsl/model/HttpHeaderRange has been compiled by a more recent version of the Java Runtime (class file version 55.0), this version of the Java Runtime only recognizes class file versions up to 52.0
```

0.3.1 is fine.

**Comment by guersam** [comment]
Thanks for the quick response, as well as this useful library :)

**Comment by lomigmegard** [comment]
You're welcome :)

Should be fixed in [0.3.3](https://github.com/lomigmegard/akka-http-cors/releases/tag/v0.3.3), the release needs some time to be propagated from Sonatype to Maven Central.
